cmd/bucket-handlers.go

```diff
@@ -566,6 +566,29 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 		return
 	}
 
+	if objectAPI.IsEncryptionSupported() {
+		if IsSSECustomerRequest(formValues) && !hasSuffix(object, slashSeparator) { // handle SSE-C requests
```

Comment:
you want to name this as `IsSSECustomerHeader` ?

Reply:
> you want to name this as IsSSECustomerHeader ?

I renamed to hasSSECustomerHeader. What do you think ?